omise/test/test_charge.py

```diff
@@ -736,6 +737,69 @@ def test_capture(self, api_call):
             'https://api.omise.co/charges/chrg_test/capture',
         )
 
+    @mock.patch('requests.post')
+    def test_capture_with_args(self, api_call):
+        charge = self._makeOne()
+        class_ = self._getTargetClass()
+        self.mockResponse(api_call, """{
+            "object": "charge",
+            "id": "chrg_test",
+            "livemode": false,
+            "location": "/charges/chrg_test",
+            "amount": 100000,
+            "captured_amount": 50000,
```

Comment:
@yurasavin can you please also add these parameters as they are a part of the response from a `get` request for a partially captured charge
- `"authorization_type": "pre_auth"`
- `"authorized_amount": 100000`